talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -82,6 +88,45 @@ void P2PClient::RemoveAllowedRemoteId(const std::string& target_id,
   }
   Stop(target_id, on_success, on_failure);
 }
+
+void P2PClient::PublishBatch(const std::string& target_id,
+               std::vector<std::shared_ptr<owt::base::LocalStream>> streams,
+               std::function<void(std::shared_ptr<P2PPublication>, std::string)> on_success,
+               std::function<void(std::unique_ptr<Exception>)> on_failure)
+{
+  // Firstly check whether target_id is in the allowed_remote_ids_ list.
```

Comment:
Both batch methods are taken from their respective unbatched versions and modified to support batching.